src/i18n/Localizable.xcstrings

```diff
@@ -10809,6 +10872,7 @@
       }
     },
     "How_To" : {
+      "comment" : "any usage example of this word? the translated one is a bit odd",
```

Comment:
"How To" is a title of a feature not yet available inside WebSSH. It's related to WebSSH Brain, which will give users steps to do something.